### PR TITLE
Unsorted header sorting, round 1

### PR DESCRIPTION
**Pull request by HeartPiece44**
Sorts a large amount of `unsorted-include-todos` and makes headers for about half of `efx`. Round 1 of n, will PR in chunks.

### CODE REVIEW

File: include/efx/TDopingSmoke.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+namespace efx {
+struct TDopingSmoke : public TSimple1 {
+	virtual void create(Arg*); // _08
+
+	// _00      = VTBL
+	// _00-_0C  = TSimple1
+};
+} // namespace efx
+
+#endif
```
Comment (EpochFlame) [inline include/efx/TDopingSmoke.h:15]: please append newlines to files for the sake of appeasing diff tools

**Comment by jonathonbrady** [comment]
lgtm (i did not look at a single file)

**Review by EpochFlame — COMMENTED** [review]
looks okay ish. have looked at several (many) files. lugia should be more diligent

**Review by EpochFlame — APPROVED** [review]
this pr: become part of the repo kthx

**Pull request merged**